Support/CMake/Toolchain-Android-X86.cmake

```diff
@@ -13,6 +13,8 @@ set(CMAKE_SYSTEM_PROCESSOR X86)
 
 set(CMAKE_C_COMPILER /tmp/aosp-toolchain/x86_64-linux-android-4.9/bin/x86_64-linux-android-gcc)
 set(CMAKE_CXX_COMPILER /tmp/aosp-toolchain/x86_64-linux-android-4.9/bin/x86_64-linux-android-g++)
+set(CMAKE_C_COMPILER_ARG1 -m32)
+set(CMAKE_CXX_COMPILER_ARG1 -m32)
```

Comment:
How about set(triple <blah>)? Seems less hacky, would go *before* the set(CMAKE_C_COMPILER <blah>) call in the file.